pyproject.toml

```
@@ -38,126 +17,129 @@ classifiers = [
   "Topic :: Scientific/Engineering",
   "Topic :: Security",
 ]
+keywords = [
+  "differential privacy",
+]
 
-# The version field is required in this file format, even though it's ignored
-# because of poetry-dynamic-versioning.
-version = "0.0.0"
-
+requires-python = ">=3.9,<4.0"
+dependencies = [
+    "numpy >=1.23.2,<2 ; python_version < '3.12'",
+    "numpy >=1.26.0,<2 ; python_version >= '3.12'",
+    "pandas >=1.4.0,<2 ; python_version < '3.11'",
+    "pandas >=1.5.0,<2 ; python_version == '3.11'",
+    "pandas >=2.2.0,<3 ; python_version >= '3.12'",
+    "pyarrow >=18.0.0,<19",
+    "pyspark[sql] >=3.3.1,<3.6 ; python_version < '3.11'",
+    "pyspark[sql] >=3.4.0,<3.6 ; python_version == '3.11'",
+    "pyspark[sql] >=3.5.0,<3.6 ; python_version >= '3.12'",
+    "randomgen >=1.20.0,<=1.26.0 ; python_version < '3.10'",
+    "randomgen >=1.23.0,<=1.26.0 ; python_version == '3.10'",
+    "randomgen >=1.26.0,<=2 ; python_version >= '3.11'",
+    "scipy >=1.6.0,<2.0.0 ; python_version < '3.10'",
+    "scipy >=1.8.0,<2.0.0 ; python_version == '3.10'",
+    "scipy >=1.9.2,<2.0.0 ; python_version == '3.11'",
+    "scipy >=1.11.2,<2.0.0 ; python_version >= '3.12'",
+    "sympy >=1.8,<1.13",
+    "typeguard >=4.0.0,<5",
+    # TODO: These should potentially be pulled out into an extra -- normal
+    # operation of Core doesn't depend on them.
+    "parameterized >=0.7.4,<0.8",
+    "pytest >=7.3.1,<8",
+    # This works around a bug in some older versions of PySpark. We should
+    # determine what versions are affected and, when possible, remove the extra
+    # dependency.
+    "setuptools >=70.0.0",
+]
 
-[tool.poetry.requires-plugins]
-poetry-dynamic-versioning = { version = ">=1.0.0,<2.0.0", extras = ["plugin"] }
+[project.urls]
+documentation = "https://docs.tmlt.dev/core/latest"
```

```suggestion
documentation = "https://tmlt.dev/core/latest"
```

docs.tmlt.dev isn't working at the moment, but tmlt.dev is.

---

Fixed.